pinot-common/src/test/java/org/apache/pinot/common/utils/PinotDataTypeTest.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.pinot.common.utils;
 
+import com.google.gson.JsonParser;
```

Comment:
Within Pinot, we try to unify the usage of json library to `fasterxml.jackson`. You may use `JsonUtils.stringToJsonNode()` to convert the string to JsonNode and then compare.

Reply:
Okay, I will update the PR with JsonUtils.